app/src/test/java/com/marekmacko/kotlinapp/data/WeatherTest.kt

```diff
@@ -0,0 +1,16 @@
+package com.marekmacko.kotlinapp.data
+
+import com.marekmacko.kotlinapp.MockData
+import com.marekmacko.kotlinapp.data.response.Weather
+import org.junit.Test
+
+import org.junit.Assert.*
+
+class WeatherTest {
+
+    @Test
+    fun getIconUrl() {
+        val weather = Weather(MockData.DESCRIPTION, MockData.ICON_CODE)
```

Comment:
It is better to keep 3 parts (or 2) in tests GIVEN WHEN THEN. Think you should test also some invalid data like empty and/or null input values. 

This comment is valid for all tests :)